<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>AlarmSuspendWidget</class>
 <widget class="QWidget" name="AlarmSuspendWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>221</width>
    <height>345</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="ToolPushButton" name="alarmPushButton">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>100</width>
     <height>100</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border-image: url(:/BMS_Center/Image/SOC/60.png);</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QTableWidget" name="alarmTableWidget">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>101</y>
     <width>220</width>
     <height>200</height>
    </rect>
   </property>
  </widget>
  <widget class="ToolPushButton" name="exclPushButton">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>70</y>
     <width>40</width>
     <height>40</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border-image: url(:/BMS_Center/Image/SOC/exclamation.png);</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
 </widget>
 <customwidgets>
  <customwidget>
   <class>ToolPushButton</class>
   <extends>QPushButton</extends>
   <header>Tools/ToolPushButton.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
